assets/img/favicon/site.webmanifest

```diff
@@ -1,14 +1,15 @@
 {
     "name": "Croydon Council",
     "short_name": "Croydon Council",
+    "start_url": "https://www.croydon.gov.uk/",
```

Comment:
Hi Chris,
This probably just need to say ```"."``` rather than "https://www.croydon.gov.uk/".  Chromium's dev tool is complaining about this start_url.  Please have a look at the [MDN example](https://developer.mozilla.org/en-US/docs/Web/Manifest#example_manifest).

On the upside, Chromium is now happy with the icon URLs.  So we are nearly there :D

Reply:
yes i was in two minds about this - but you are right - eventually it will be on this domain and everything will be ok.